src/tests/ftest/daos_test/dfuse.py

```diff
@@ -91,8 +91,15 @@ def run_test(self, il_lib=None):
             daos_test_env['DD_MASK'] = 'all'
             daos_test_env['DD_SUBSYS'] = 'all'
             daos_test_env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
-
-        command = [self.daos_test, '--test-dir', mount_dir, '--io', '--stream']
+            if il_lib == 'libpil4dfs.so':
+                daos_test_env['D_IL_REPORT'] = '0'
+                daos_test_env['DAOS_MOUNT_POINT'] = f'{mount_dir}'
+                daos_test_env['DAOS_POOL'] = f'{self.pool.identifier}'
+                daos_test_env['DAOS_CONTAINER'] = f'{self.container.uuid}'
```

Comment:
No need to use f-strings for these. Also, any reason why the container label wouldn't work?
```suggestion
                daos_test_env['DAOS_MOUNT_POINT'] = mount_dir
                daos_test_env['DAOS_POOL'] = self.pool.identifier
                daos_test_env['DAOS_CONTAINER'] = self.container.identifier
```

Reply:
Container label should work. I will fix them. Thank you very much!

Reply:
Fixed as you suggested. Thank you!